plotter/db.py

```diff
@@ -581,12 +650,30 @@ def upper_thresh(copies):
 http://bioviz.rocq.inria.fr/export/%(user)s/%(name)s/copies/bed/
 """
 
+def reprocess_all_profiles():
```

Comment:
this `reprocess_all_profiles` method will be useful if we want to use your new chrom viewer images with an existing SegAnnDB server